src/common/parse/spec/multilineTrim.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {multilineTrim} from '../multilineTrim';
+
+describe('MultilineTrim', () => {
+  it('is defined', () =>{
+    //Arrange
+    //Act
+    //Assert
+    expect(multilineTrim).not.to.be.undefined;
+  });
+
+  it ('should return a string', () => {
+    //Arrange
+    const string = '';
+    //Act
+    //Assert
+    expect(multilineTrim(string)).to.be.a('string');
+  });
+
+  it('should trim new lines', () => {
+    //Arrange
+    const string = `
+    <tr>
+      <td>Hi</td>
+      <td>Hello</td>
+      <td>Arr!!</td>
+    </tr>
+    `;
+    //Act
+    //Assert
+    expect(multilineTrim(string)).to.be.equals('<tr><td>Hi</td><td>Hello</td><td>Arr!!</td></tr>');
+  });
+
+  it('should preserve spaces inside lines', () => {
+    //Arrange
+    const string = `
+    <tr>
+      <td>John Doe</td>
+      <td>Mark Flow</td>
+    </tr>
+    `;
+    //Act
+    //Assert
+    expect(multilineTrim(string)).to.be.equals('<tr><td>John Doe</td><td>Mark Flow</td></tr>');
+  });
+
```

Comment:
Pending testing attributes spaces

```
<p style="color: black">
</p> 
```

What about text literals?

```
<span>
my test with spaces
</span>
```

What would happen if we have some spaces and white lines?

```
<span>
my test with spaces
</span>
   

<span>
</span>
```